server/game/basecard.js

```diff
@@ -170,15 +171,14 @@ class BaseCard {
         this.abilities.reactions.push(reaction);
     }
 
-    // TODO: When revealed abilities shouldn't be a synonym for forced reactions
-    //       but it is probably close enough for now.
     whenRevealed(properties) {
```

Comment:
Correct, I just lumped them all in one spot when building out the ability API. Could certainly move it to `PlotCard`.